packages/core/src/app/payment/StoreInstrumentFieldset/InstrumentStoreAsDefaultField.tsx

```diff
@@ -7,16 +8,25 @@ import { CheckboxFormField } from '../../ui/form';
 interface InstrumentStoreAsDefaultFieldProps {
     isAccountInstrument: boolean;
     disabled?: boolean;
+    formik?: Formik;
 }
 
 const InstrumentStoreAsDefaultField: FunctionComponent<InstrumentStoreAsDefaultFieldProps> = ({
     isAccountInstrument,
     disabled = false,
+    formik,
 }) => {
     const translationId = isAccountInstrument
         ? 'payment.account_instrument_save_as_default_payment_method_label'
         : 'payment.instrument_save_as_default_payment_method_label';
 
+    useEffect(() => {
```

Comment:
Instead of updating this can we try updating https://github.com/bigcommerce/checkout-js/blob/master/packages/instrument-utils/src/storedInstrument/InstrumentStoreAsDefaultField/InstrumentStoreAsDefaultField.tsx and using this one. Also same for the `StoreInstrumentFieldset` file.